gbn/config.go

```diff
@@ -2,6 +2,69 @@ package gbn
 
 import "time"
 
+// TimeoutOptions can be used to modify the default timeout values used within
+// the TimeoutManager.
+type TimeoutOptions func(manager *TimeoutManager)
+
+// WithResendTimeout is used to set a static resend timeout. This is the time to
+// wait for ACKs before resending the queue.
+func WithResendTimeout(timeout time.Duration) TimeoutOptions {
+	return func(manager *TimeoutManager) {
+		manager.SetStaticResendTimeout(timeout)
```

Comment:
can just set the value directly. Dont need to go through the "Set*" methods

Reply:
See https://github.com/lightninglabs/lightning-node-connect/pull/88#discussion_r1429309650

Reply:
Addressed with the latest push, thanks!